modules/analytics/src/views/full/ItemsList.tsx

```diff
@@ -28,7 +28,7 @@ const ItemsList: FC<Props> = props => {
       <ol>
         {items.map((item, index) => (
           <li key={index}>
-            <a onClick={item.onClick} className={!item.onClick && style.disabled}>
+            <a onClick={item.onClick} className={item.onClick ? undefined : style.disabled}>
```

Comment:
Please use classnames for conditional styling: `className={cx({ [style.disabled]: !!item.onClick })}`